src/System.Private.ServiceModel/tests/Scenarios/Bridge/WcfTestBridgeCommon/Properties/AssemblyInfo.cs

```diff
@@ -0,0 +1,36 @@
+﻿using System.Reflection;
+using System.Runtime.CompilerServices;
+using System.Runtime.InteropServices;
+
+// General Information about an assembly is controlled through the following 
+// set of attributes. Change these attribute values to modify the information
+// associated with an assembly.
+[assembly: AssemblyTitle("WcfTestBridgeCommon")]
+[assembly: AssemblyDescription("")]
+[assembly: AssemblyConfiguration("")]
+[assembly: AssemblyCompany("Microsoft Corporation")]
+[assembly: AssemblyProduct("WcfTestBridgeCommon")]
+[assembly: AssemblyCopyright("Copyright © Microsoft Corporation 2015")]
+[assembly: AssemblyTrademark("")]
+[assembly: AssemblyCulture("")]
```

Comment:
Ditto, probably best to get rid of anything copyright-related in this file. If you look at equivalent files in corefx, they are minimal, e.g.: https://github.com/dotnet/corefx/blob/fe33ff4bcbe4a3b4b47854388e68358a29bac93d/src/System.Dynamic.Runtime/src/Properties/AssemblyInfo.cs